AstSemantics.md

```diff
@@ -96,107 +120,61 @@ following rules:
 Wrapping of integers simply discards any upper bits; i.e. wrapping does not
 perform saturation, trap on overflow, etc.
 
-## Addressing local variables
+In addition to storing a value to linear memory, store instructions also return
+their value operand, with no conversion applied.
```

Comment:
"Store expressions" and "produce a value" rather than return, since we want to avoid confusion about returning from the function.